apps/health_campaign_field_worker_app/integration_test/attendance_test.dart

```diff
@@ -0,0 +1,217 @@
+import 'package:health_campaign_field_worker_app/widgets/home/home_item_card.dart';
+import 'package:integration_test/integration_test.dart';
+import 'package:hrk_flutter_test_batteries/hrk_flutter_test_batteries.dart';
+import './test_variables.dart';
+
+void main() {
+  IntegrationTestWidgetsFlutterBinding.ensureInitialized();
+
+  final _username = find.byKey(const Key('username'));
+  final _password = find.byKey(const Key('password'));
+
+  final testVariables = getTestData();
+
+  testWidgets('Get to the manage Attendance Page', (widgetTester) async {
+    disableOverflowError();
+    app.main();
+
+    await widgetTester.pumpAndSettle(
+      const Duration(seconds: 5),
+    ); //wait for the app to load languages
+
+    await widgetTester
+        .tap(find.widgetWithText(DigitElevatedButton, 'Continue'));
+
+    await widgetTester.pumpAndSettle(
+      const Duration(milliseconds: 500),
+    ); //wait for the app to load the login page
+
+    expect(
+      find.byType(LoginPage),
+      findsOneWidget,
+    ); //check if the login page is loaded
+
+    await widgetTester.enterText(
+      _username,
+      testVariables['username'],
+    ); //enter the username to test
+    await widgetTester.enterText(
+      _password,
+      testVariables['password'],
+    ); //enter the password to test
+
+    await widgetTester
+        .tap(find.byKey(const Key('login'))); //tap on the login button
+    await widgetTester.pumpAndSettle(const Duration(seconds: 5));
+
+    expect(
+      find.byType(BoundarySelectionPage),
+      findsOneWidget,
+    ); //check if the boundary page is loaded
+
+    //select the boundaries
+    List<String> boundaryNames = [
+      'MOÇAMBIQUE',
+      'NAMPULA',
+      'MURRUPULA',
+      'NIHESSIUE',
+      'NIHESSIUE',
+      'CAVINA1',
+    ];
+    await widgetTester.pumpAndSettle(const Duration(seconds: 2));
+    for (int i = 0; i < 6; i++) {
+      await widgetTester
+          .tap(find.byType(DigitReactiveSearchDropdown<BoundaryModel>).at(i));
+      await widgetTester.pumpAndSettle(const Duration(milliseconds: 100));
+      await widgetTester.tap(find.text(boundaryNames[i]).last);
+      await widgetTester.pumpAndSettle(const Duration(milliseconds: 100));
+    }
+
+    //submit
+    await widgetTester.tap(find.widgetWithText(
+      DigitElevatedButton,
+      'Submit',
+    ));
+
+    //go to home page after boundary selection
+    await widgetTester.pumpAndSettle(const Duration(seconds: 1));
+    expect(find.byType(HomePage), findsOneWidget);
+
+    //tap on manage attendance
+    await widgetTester
+        .tap(find.widgetWithText(HomeItemCard, 'Manage Attendance'));
+
+    await widgetTester.pumpAndSettle(const Duration(seconds: 1));
+
+    //check if we are on the manage attendance page
+    expect(
+      find.byType(ManageAttendancePage),
+      findsOneWidget,
+      reason: 'Manage Attendance Page not found',
+    );
+
+    expect(find.text('Open Register'), findsAtLeast(1));
```

Comment:
Can check digit elevated button instead of text 
Reason: In future if button text is changed we have to change the test case as well.